pkgs/top-level/all-packages.nix

```diff
@@ -42065,4 +42065,6 @@ with pkgs;
   insulator2 = callPackage ../applications/misc/insulator2 {};
 
   animdl = python3Packages.callPackage ../applications/video/animdl { };
+
+  gow = callPackage ../by-name/go/gow { };
```

Comment:
When using the new `by-name` convention, you don't need to add the package to this file unless it requires some explicit arguments and/or different call namespace.
```suggestion
```